tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -637,3 +637,10 @@ test_i2c_max31875: max31875@53 {
 	compatible = "maxim,max31875";
 	reg = <0x53>;
 };
+
+test_i2c_s11059: s11059@2a {
+	compatible = "hamamatsu,s11059";
+	reg = <0x2A>;
```

Comment:
Change the address to `54`

Reply:
This device is actually working with address that is 0x2a (0x54 is not working).
The document describes the value after shifting 1bit of R/W from the 7bit address, and it is written in a confusing way. (That is, 8bit address!)

Reply:
The address in this file does not need to fit the real world address. As you can read at the top of the File, the address is held sequential over the whole file. Take a look on the other entries in this file.

Reply:
@pefech 
Thank you for your review.
I made the fix as per your advice.